### DISCUSSION TITLE
ON / OFF CMD for ST7735 Display

### DISCUSSION THREAD

**Discussion by Patschifig2021** [Bodmer/TFT_eSPI#1196] [Q&A - General]
Dear all
For my project I have two 0,96inch Displays connected to one ESP32. I was wondering if there was a simple command/way to switch the displays ON or OFF (Energy saving is not really a factor as I am supplying my installation using USB).
Thanks for your help and ideas.
Remo

**Comment by Bodmer** [comment]
The big power consumer is usually the backlight LEDs so adding a transistor to turn them on/off may be an option if the LED power connections are available.
